pkg/component/worker/kubelet.go

```diff
@@ -170,15 +191,21 @@ func (k *Kubelet) Start(ctx context.Context) error {
 		args["--node-labels"] = strings.Join(k.Labels, ",")
 	}
 
+	nodename, err := node.GetNodename("")
+	if err != nil {
+		return fmt.Errorf("can't get hostname: %v", err)
+	}
+
+	ipv4, ipv6, err := getIPs(nodename)
+	if err == nil && ipv4 != nil && ipv6 != nil {
+		args["--node-ip"] = ipv4.String() + "," + ipv6.String()
```

Comment:
Do we want to set this unconditionally, or maybe only when k0s is running in dual stack mode?

Reply:
After second thought, we probably want hide it behind k0s dualstack mode